packages/backend/src/core/activitypub/ApRendererService.ts

```diff
@@ -466,14 +466,24 @@ export class ApRendererService {
 			this.userProfilesRepository.findOneByOrFail({ userId: user.id }),
 		]);
 
-		const attachment = profile.fields.map(field => ({
+		const profileFields = profile.fields.map(field => ({
 			type: 'PropertyValue',
 			name: field.name,
 			value: (field.value.startsWith('http://') || field.value.startsWith('https://'))
 				? `<a href="${new URL(field.value).href}" rel="me nofollow noopener" target="_blank">${new URL(field.value).href}</a>`
 				: field.value,
 		}));
 
+		const mutualLinks = profile.mutualLinkSections.flatMap(section =>
+			section.mutualLinks.map(link => ({
+				type: 'PropertyValue',
+				name: (section.name ?? link.description) as string,
```

Comment:
```suggestion
				name: (section.name ?? link.description ?? 'Link') as string,
```
セクションの名前もリンクの説明文もoptionalなフィールドなので、何れもfalsyの場合のテキストを定義する必要がありそう
（少なくともMisskeyの実装ではfieldsのnameとvalueが空白ではない文字列であることを強制している）